WcaOnRails/app/webpacker/components/Panel/PanelList.js

```diff
@@ -0,0 +1,23 @@
+const PanelList = {
```

Comment:
I think this should be `panelList` (or maybe `PANEL_LIST`?).

Reply:
`PANEL_LIST` looks better. Thank you Kevin.